apps/verticalhome/index.html

```diff
@@ -68,50 +73,56 @@
     <link rel="stylesheet" href="app://theme.gaiamobile.org/shared/elements/gaia-theme/gaia-theme.css">
     <!-- <script defer src="shared/elements/gaia-header/dist/gaia-header.js"></script> -->
 
+    <script defer src="js/pin_app_navigation.js"></script>
+    <script defer src="js/more_apps_navigation.js"></script>
+    <script defer src="js/clock.js"></script>
     <script defer src="js/app.js"></script>
-    <script defer src="js/search.js"></script>
+<!--    <script defer src="js/search.js"></script> -->
```

Comment:
do we need 'search.js' script in verticalhome app? If no, then it's better to delete this line instead of commenting, since commented scripts will be packaged into 'application.zip' archive and will be available for loading from other scripts. (AFAIR so please correct me if I'm wrong)